api/PayrollProcessor.Functions.Api/Features/Departments/DepartmentEmployeeTrigger.cs

```diff
@@ -77,7 +78,10 @@ public class DepartmentEmployeeTrigger
                     .Bind(
                         employee => commandDispatcher.Dispatch(new DepartmentEmployeeCreateCommand(employee, idGenerator.Generate()))
                             .DoIfFail(ex => log.LogError(ex, "Could not create Department Employee for {@employee}", employee))
-                            .Do(de => log.LogInformation("{@departmentEmployee} created for {employeeId}", de, message.EmployeeId))))
+                            .Do(de => log.LogInformation("{@departmentEmployee} created for {employeeId}", de, message.EmployeeId))
+                            .ToTryOption()
+                    )
```

Comment:
Not sure if having the parens match with the method calls in indentation is better or if having them all line up at the end of the last call is better (see below)